pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[project]
+name = "cumulus-fhir-support"
+requires-python = ">= 3.9"
+dependencies = [
+    "fhirclient >= 4.1",
+    "pyarrow >= 12",
+]
+authors = [
+  { name="Michael Terry", email="michael.terry@childrens.harvard.edu" },
+]
+description = "FHIR schema support code for the Cumulus project"
+readme = "README.md"
+license = { text="Apache License 2.0" }
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dynamic = ["version"]
+
+[project.urls]
+"Homepage" = "https://github.com/smart-on-fhir/cumulus-fhir-support"
+
+[build-system]
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.sdist]
+include = [
+    "tests/",
+    "LICENSE",
+]
+
+[tool.bandit]
+exclude_dirs = ["tests"]
+
+[tool.black]
+line-length = 100
```

Comment:
I'm proposing 100 as a future compromise length, but no need to agree now - I'm happy to experiment with it